docs/useCanAccessCallback.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: default
+title: "useCanAccessCallback"
+---
+
+# `useCanAccessCallback`
+
+This hook returns a callback to call the `authProvider.canAccess()` method for a provided resource and action (and optionally a record).
+
+## Usage
+
+`useCanAccessCallback` returns an async function that takes an object `{ action, resource, record }` as argument. This function returns an object with the following properties:
+
+- `canAccess`: a boolean determining whether the user has access to the resource.
+- `error`: the error that might have occurred.
```

Comment:
This API is strange. I'd expect it to return `canAccess` directly and throw in case of error. Why return an object instead?